tc_site/urls.py

```diff
@@ -17,5 +17,6 @@
     path('gen-file/', views.gen_file, name="gen-file"),
     path('download/<int:userID>/<int:docID>', views.download, name="download"),
     path('share/<int:userID>/<int:docID>', views.share, name="share"),
-    path('export/<int:userID>/<int:docID>', views.export, name="export"),
+    # path('export/<int:userID>/<int:docID>', views.export, name="export"),
+    path('export/', views.export, name="export"),
```

Comment:
Those url slugs need to be there because they will be use to get a particular document from the database. All you need to do is accept `userID` and `docID` and parameters to your view helper function and then use them in it.

Reply:
yeah this was just to show you how the code works if it was to be exported from a html file to pdf.
I'll work on using `ID` now